rights-statements.ttl

```diff
@@ -28,202 +30,216 @@
 
 <collection-ic> a skos:Collection ;
   skos:prefLabel "rights statements for works that are in copyright"@en ;
-  skos:member <InC> ;
-  skos:member <InC-OW-EU> ;
-  skos:member <InC-RUU> ;
-  skos:member <InC-EDU> ;
-  skos:member <InC-NC> .
+  skos:member <InC/1.0/> ;
+  skos:member <InC-OW-EU/1.0/> ;
+  skos:member <InC-RUU/1.0/> ;
+  skos:member <InC-EDU/1.0/> ;
+  skos:member <InC-NC/1.0/> .
 
 <collection-pd> a skos:Collection ;
   skos:prefLabel "rights statements for works that are not in copyright"@en ;
-  skos:member <OOC-NC> ;
-  skos:member <NoC-CR> ;
-  skos:member <NoC-OLR> .
+  skos:member <OOC-NC/1.0/> ;
+  skos:member <NoC-CR/1.0/> ;
+  skos:member <NoC-OKLR/1.0/> .
 
 <collection-other> a skos:Collection ;
   skos:prefLabel "other rights statements"@en ;
-  skos:member <NKC> ;
-  skos:member <CNE> .
+  skos:member <NKC/1.0/> ;
+  skos:member <CNE/1.0/> .
 
 # Individual statements
 # 2015-07-15: updated from https://docs.google.com/document/d/1tdRAFZZefiyygRwCbcUq-kK7mNdj-6UFwBSCrPRSIoY/edit
 # 2015-07-17: updated from document to use correct URI patterns
 # 2015-08-11: update to make individual statements skos:Concepts, and to 
 #             use skos:note to model the Notices
+# 2015-09-16: updated text to match detailed Rights Statements document
```

Comment:
Please add a note that you've commented out `OOC-NC` - I don't want us to lose track of that